tool/gravity/cli/system.go

```diff
@@ -1276,7 +1276,7 @@ func unpack(p *localpack.PackageServer, loc loc.Locator) error {
 	if err != nil {
 		return trace.Wrap(err)
 	}
-	return trace.Wrap(pack.Unpack(p, loc, path, nil))
+	return p.Unpack(loc, path)
```

Comment:
I'm not sure why this is still here (in cli package) since upgrade uses lib/update/system now but decided to update it here as well just in case. IIRC we kept this to support legacy "gravity system update" command, which we should ideally rewrite to use lib/ too.